### ISSUE TITLE
butchery fails are not being recognized

### ISSUE THREAD

**Issue by halprind** [rpherbig/dr-scripts#2555]
```
[combat-trainer]>drop my
Please rephrase that command.
>
[combat-trainer]>perform butcher on wyvern
You notice Insydious slip into a hiding place.
>
This ritual may only be performed on a creature's corpse.
>
[combat-trainer]>drop my
Please rephrase that command.
>
[combat-trainer]>perform butcher on wyvern
This ritual may only be performed on a creature's corpse.
>
[combat-trainer]>drop my
Please rephrase that command.
>
[combat-trainer]>perform butcher on wyvern
This ritual may only be performed on a creature's corpse.
>
[combat-trainer]>drop my
Please rephrase that command.
>
[combat-trainer]>perform butcher on wyvern
This ritual may only be performed on a creature's corpse.
>
[combat-trainer]>drop my
Please rephrase that command.
>
[combat-trainer]>perform butcher on wyvern
This ritual may only be performed on a creature's corpse.
>
[combat-trainer]>drop my
Please rephrase that command.
>
[combat-trainer]>perform butcher on wyvern
This ritual may only be performed on a creature's corpse.
>
[combat-trainer]>drop my
Appearing to have lost sight of its target, a young wyvern stops advancing.
```

base-spells has this:
```
failures:
  - don't have enough thanatological
  - You cannot harvest
  - Rituals do not work upon the undead
  - This ritual may only be performed on
  - This corpse has already been
  - roundtime
  - A failed or completed ritual has rendered this corpse unusable for your purpose
  - unable to discern where to direct your thanatological expertise
  - You realize the unfortunate butchery ruins the body for any further attempts at rituals
  - A skinned creature is worthless for your purposes
  - The consumption ritual performed on this corpse prevents its `harvesting
```

**Comment by izawwlgood** [comment]
I've also seen this a few times, and I too have no idea why it's happening. It seems to only happen when someone else is in the room.

**Comment by PatricCampbell** [comment]
I had a similar one after healing myself on the corpse:

`perform butcher on giant

The consumption ritual performed on this corpse renders it useless for butchering.`

**Comment by izawwlgood** [comment]
Maybe 'useless for butchering' will be a sufficient catch all?

**Comment by rpherbig** [comment]
This should be fixed by #2557
